src/Components/Buttons.jsx

```diff
@@ -12,25 +12,27 @@ import {assertDefined} from '../utils/assert'
  * @param {Object} icon
  * @param {string} placement
  * @param {string} size
+ * @param {string} dataTestId Internal attribute for component testing
  * @return {Object} React component
  */
-export function TooltipIconButton({
+export const TooltipIconButton = ({
```

Comment:
So we were using more modern syntax but there was something about function that worked better.  I forget what.  Is there a reason to prefer const here besides shorter?

Reply:
I suggest reverting this to function as it's more consistent with the rest of our code.  I think the rule we're using is "top-level exported functions should use function keyword".  I'll add that to styleguide.

Reply:
https://github.com/bldrs-ai/Share/wiki/Style-Guide